common/app/pagepresser/InteractiveImmersiveHtmlCleaner.scala

```diff
@@ -0,0 +1,73 @@
+package pagepresser
```

Comment:
I've moved the pagepresser cleaners into common - hope this is ok? Moving them into common allowed me to:
- create a new cleaner that extends from the common class (HtmlCleaner)
- import this package in the interactive librarian, which lives in common as opposed to admin

Reply:
👍 this makes sense to me